### PR TITLE
Client

### PR DESCRIPTION
**Pull request by grahamdyson**

### CODE REVIEW

File: restricted/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (cheryllaird) [inline restricted/index.html:1]: I can navigate to this page as it is not currently auth protected. Can you set it up so this functionality is working please?

File: login/login.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	};
+}
+
+function handleResponse(
+	response,
+) {
+	if (response.ok)
+		location.href = "/";
+	else
+		setErrorText(
```
Comment (cheryllaird) [inline login/login.js:31]: Where is `setErrorText` defined?

File: login/login.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+function login() {
+	if (document.getElementById("login-form").reportValidity())
+		fetch(
```
Comment (cheryllaird) [inline login/login.js:3]: The API call is not working at the moment, can we stub this for now and get it working with some test credentials?

**Review by cheryllaird — COMMENTED** [review]
It looks like we have the basic markup and login functions set up but it isn't functional at the moment. Can we stub the api while the server PR is being approved and get the app working. I think lets get it to a point where we can login with some test credentials and the app to display if we are a logged in user or not.

File: login/login.js (hunk 2 above)
Comment (grahamdyson) [reply login/login.js:31]: i made a with mistake with the name of the call and fixed it in commit 04b2b9b, but into the server branch. i should have done made that fix in this branch instead. i can cherry pick that commit from server and bring it into this branch/pull request.

**Comment by grahamdyson** [comment]
The server branch only adds a stub with hardcoded test credentials and an example of a access restricted page. As I did the client first I made two branches, with things like css customisability and accessibility shown here, but might be better to just look at both client and server together in the server pull request.

**Pull request closed without merging**